src/components/RadioGroup/RadioGroup.jsx

```diff
@@ -0,0 +1,143 @@
+import _ from 'lodash';
+import classNames from 'classnames';
+import React from 'react';
+
+import { bindClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition } from '../../util/component-definition';
+import getRandom from '../../util/random';
+
+import RadioButton from '../RadioButton/RadioButton';
+import reducers from './RadioGroup.reducers';
+
+const boundClassNames = bindClassNames('RadioGroup');
+
+const {
+	arrayOf,
+	func,
+	node,
+	number,
+	oneOfType,
+	string
+} = React.PropTypes;
+
+/**
+ * {"categories": ["controls", "toggles"]}
+ *
+ * This is a group of related radio buttons whose values are mutually exclusive
+ * and one whom must be selected any given moment in time.
+
+ * Any props that are not explicitly defined in `propTypes` are spread onto the
+ * root element.
+ */
+const RadioGroup = React.createClass(createLucidComponentDefinition({
+	displayName: 'RadioGroup',
+
+	childProps: {
+		RadioButton: { ...RadioButton.propTypes }
+	},
+
+	reducers,
+
+	propTypes: {
+		/**
+		 * Should be instances of `RadioGroup.RadioButton` which supports the
+		 * same props as `RadioButton`.
+		 */
+		children: oneOfType([
+			arrayOf(node),
+			node
```

Comment:
I don't think that's _technically_ true insofar as what React actually produces in its renders.

```
<RadioGroup name={name}>
	<RadioGroup.RadioButton />
	<RadioGroup.RadioButton />
	<RadioGroup.RadioButton />
</RadioGroup>
```

This results in a `RadioGroup` node that has three `RadioGroup.RadioButton` nodes stored in its `nodes` array.

It's not possible to just render the following:

```
<RadioGroup.RadioButton />
<RadioGroup.RadioButton />
<RadioGroup.RadioButton />
```

But if it were it would result not in a single node entity but rather an array of them.

With all that said, though, the documentation for `React.PropTypes` _does_ note that when we use `React.PropTypes.node` that check also includes arrays of the represented types...

...so, yeah, I'll simplify that statement. :)